kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -54,7 +54,15 @@ data:
           # Requires ContainerLogV2 schema to be enabled. See https://aka.ms/ContainerLogv2 for more details.
           # enabled = "false"
           # stacktrace_languages = []
-
+       #[log_collection_settings.metadata_collection]
+          # kube_meta_cache_ttl is a configurable option for K8s cached metadata. Default is 60s. You may adjust it in below section [agent_settings.k8s_metadata_config]. Reference link: https://docs.fluentbit.io/manual/pipeline/filters/kubernetes#configuration-parameters
+          # if enabled will collect kubernetes metadata for ContainerLogv2 schema. Default is false. 
+          # enabled = false
+          # if include_fields commented out or empty, all fields will be included. If include_fields is set, only the fields listed will be included.
+          # include_fields = ["podLabels","podAnnotations","podUid","image","imageID","imageRepo","imageTag"]
+       #[log_collection_settings.filter_using_annotations]
```

Comment:
@pfrcks , we would need to validate this works for Geneva path also since  cosmic team asking for this.